tools/editor/editor_node.cpp

```diff
@@ -1926,6 +1925,8 @@ void EditorNode::_run(bool p_current,const String& p_custom) {
 
 	List<String> breakpoints;
 	editor_data.get_editor_breakpoints(&breakpoints);
+    
+    args = Globals::get_singleton()->get("editor/main_run_args");
```

Comment:
Nothing personal (not to you nor to them), but you put some spaces instead of a tab here.